index.html

```diff
@@ -196,10 +196,26 @@ <h3 class="connect">LET'S CONNECT</h3>
               </li>
             </ul>
           </li>
-          <li class="cat-list-items">Framwork <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg" alt=""></span></li>
-          <li class="cat-list-items">Skills <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg" alt=""></span></li>
+          <li class="cat-list-items">Framwork <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg"
+                alt=""></span></li>
+          <li class="cat-list-items">Skills <span><img class="arrow right-arrow" src="src/icons/right-arrow.svg"
+                alt=""></span></li>
         </ul>
       </section>
+      <section class="contact-section">
+        <form action="" method="get" class="contact-form">
```

Comment:
- [ ] Kindly create a form with [formspree](https://formspree.io/html) services that generate a unique link that you will use in your `action` attribute so that your form can be submitted.
- [ ] Kindly note that in this instance we will be using the `POST` method and not `GET` method as we are submitting data to formspree